### PR TITLE
Passing adapter to provider model

### PR DESCRIPTION
**Pull request by dragonfly91**

### CODE REVIEW

File: src/ResourceManager/RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesContainer.cs [hunk 1]
```diff
@@ -17,22 +17,48 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Management.Automation;
```
Comment (sudhakar4171) [inline src/ResourceManager/RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesContainer.cs]: minor - looks good if you can put them in order. also in general, group by system and followed by internal

File: src/ResourceManager/RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesContainer.cs [hunk 2]
```diff
@@ -17,22 +17,48 @@
...
 using System.Text;
 using System.Threading.Tasks;
 
 namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets
 {
-    class GetAzureRmRecoveryServicesContainer : RecoveryServicesBackupCmdletBase
+    [Cmdlet(VerbsCommon.Get, "AzureRmRecoveryServicesContainer"), OutputType(typeof(List<AzureRmRecoveryServicesContainerBase>))]
```
Comment (sudhakar4171) [inline src/ResourceManager/RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesContainer.cs]: doesn't we need to specify single get returntype as well?

File: src/ResourceManager/RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesContainer.cs (hunk 1 above)
Comment (dragonfly91) [inline src/ResourceManager/RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesContainer.cs]: This was sorted by default sorting option in VS (Reorder Usings option)

Comment (AditTulasi) [inline src/ResourceManager/RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesContainer.cs]: I think what Sudhakar means is that you can put "System" related usings above our internal usings. You can configure this in VS.

**Pull request merged**